src/Autowhatever.js

```diff
@@ -235,4 +241,28 @@ export default class Autowhatever extends Component {
       </div>
     );
   }
+
+  componentDidUpdate() {
+    if (this.refs.focusedItem) {
+      const itemNode = findDOMNode(this.refs.focusedItem);
+      const containerNode = findDOMNode(this.refs.itemsContainer);
```

Comment:
I'm pretty sure that `findDOMNode` is not required anymore. Can we try to remove it and see what happens?